docs/workflows/developing-a-cumulus-workflow.md

```diff
@@ -0,0 +1,58 @@
+---
+id: developing-a-cumulus-workflow
+title: Developing A Cumulus Workflow
+hide_title: true
+---
+
+# Creating a Cumulus workflow
+
+## The Cumulus workflow module
+
+To facilitate adding a workflows to your deployment Cumulus provides a [workflow module](https://github.com/nasa/cumulus/tree/master/tf-modules/workflow).
+
+In combination with the [Cumulus message](cumulus-task-message-flow), the workflow module provides a way to easily turn a Step Function definition into a Cumulus workflow, complete with:
+
+- [AWS CloudWatch events](https://docs.aws.amazon.com/cloudwatch/index.html) that handle database updates for Cumulus objects (e.g. `executions`, `granules`, etc)
+- Built-in integration with our [throttling](../data-cookbookx/throttling-queued-executions) feature
```

Comment:
This link isn't working for me - you have an `x` in there

Reply:
4fd927328